Packs/TruSTAR/Integrations/TruSTAR/CHANGELOG.md

```diff
@@ -1,5 +1,5 @@
 ## [Unreleased]
-
+- Updated Phishing Triage calls
```

Comment:
Please add here more details, what has been changed and how it will impact the users.
This way all the customers will see release notes.
Also please mention the `Breaking changes: ` and list the them.
Something like:
```
Breaking changes:
- `normalized_triage_score ` argument replaced by `priority_event_score` in `trustar-get-phishing-submissions` command
```